antarest/study/storage/utils.py

```diff
@@ -287,28 +313,27 @@ def get_start_date(
     start_offset = cast(int, config.get("simulation.start"))
     end = cast(int, config.get("simulation.end"))
 
-    starting_month_index = strptime(starting_month, "%B").tm_mon
+    starting_month_index = ENGLISH_MONTHS.index(starting_month.title()) + 1
```

Comment:
There is no performance issue for a list of size 12 (or 7 with days).
To me the code is clear enough. The difference : we may have `IndexError` instead of `KeyError`.